### PR TITLE
Open funders

### LINKED ISSUE #114: [Funders] Integrate funding programs & orgs from open-funding
See the following repo (CC-BY-SA 4.0)

https://github.com/ralphtheninja/open-funding?tab=readme-ov-file

We will have to think about how to do attribution.

### PR DESCRIPTION
**Pull request by Nolski**
After reviewing the list of [open funders](https://github.com/ralphtheninja/open-funding) and each individual funding program, I found that the included programs are still ongoing and would be generally relevant to most academics involved in open source.

resolves #114

### CODE REVIEW

File: funders/index.md [hunk 1]
```diff
@@ -30,3 +30,125 @@ A full list is [here](https://en.wikipedia.org/wiki/Crown_Research_Institute).
 - [The Alfred P. Sloan Foundation](https://sloan.org/)
 - [Chan Zuckerberg Initiative](https://chanzuckerberg.com/)
 - [Ford Foundation](https://www.fordfoundation.org/)
+
+## Individual Funding Opportunities
+
+The below funding opportunities have been taken from the [open-funding](https://github.com/ralphtheninja/open-funding) github repository and are sub-set of funding specifically relevant to academia.
+
+### [`Protocol Labs Research`](https://github.com/protocol/research#private-retrieval)
```
Comment (RichardLitt) [inline funders/index.md:38]: Let's move private crypto to its own section, or remove it entirely.

File: funders/index.md [hunk 2]
```diff
@@ -30,3 +30,125 @@ A full list is [here](https://en.wikipedia.org/wiki/Crown_Research_Institute).
...
+
+We encourage you to reach out to `research-grants@protocol.ai` or visit `#private-retrieval` in the Lodestar Discord if you’re considering applying or have any questions.
+
+#### Deadline
+
+N/A
+
+---
+
+### [`Chan Zuckerberg Initiative`](https://chanzuckerberg.com/eoss/)
```
Comment (RichardLitt) [inline funders/index.md:65]: These all look like they should be on their own pages - what do you think about that idea?

**Review by RichardLitt — CHANGES_REQUESTED** [review]

File: funders/index.md (hunk 1 above)
Comment (Nolski) [reply funders/index.md]: @RichardLitt given they are now broken out into their own pages, I marked which funders were web3 related in the list. What do you think of that?

File: funders/czi.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+We welcome applications from any country, provided the proposed work is compliant with the United States Treasury Department’s Office of Foreign Asset Control (OFAC) sanctions program.
+Applications may be submitted by non-profit and for-profit organizations, public and private institutions, such as colleges, universities, hospitals, laboratories, units of state and local government, companies, and eligible agencies of the federal government. As part of the full application process, for-profit organizations may need to provide additional information on the charitable purposes of the proposal. Grants are not permitted to individuals; only to organizations. Open source software projects operating independently must be affiliated with an organization, as described below.
+
+## Application
+
+<https://chanzuckerberg.com/rfa/essential-open-source-software-for-science/>
+
+## Deadline
+
+Cycle 6 closed in October 2023.
```
Comment (RichardLitt) [inline funders/czi.md:18]: Is this still open, then?

File: funders/cisco.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+### Web 3
+
+The goal of this RFP is to solicit proposals that aim to conduct research and address research challenges in the technologies and applications in the area of Web 3.0, Blockchain, cryptocurrencies aka virtual-currencies. Specific areas of research include but are not limited to the following:
+
+Blockchain platforms, protocols, cross-chain protocols, consensus, smart contracts, de-centralization
+Networking, Compute, Edge, Cloud Infrastructure for Web 3.0:
+Blockchain
+De-centralized Apps (dApps)
+De-centralized Finance (DeFi)
+Trust, Security & Privacy:
```
Comment (RichardLitt) [inline funders/cisco.md:28]: Format for markdown lists.

**Review by RichardLitt — CHANGES_REQUESTED** [review]
Thanks! Small comments.

File: funders/czi.md (hunk 3 above)
Comment (Nolski) [reply funders/czi.md:18]: No but they do run future cycles so this date may indicate the next funding cycle I suppose. (This is copied from the open-funders repo)

Comment (Nolski) [reply funders/czi.md:18]: No it is closed. I think the date is there to give indication to the funding cycle.

**Comment by RichardLitt** [comment]
Merged!

**Pull request closed without merging**